src/widget/widget.cpp

```diff
@@ -498,6 +498,19 @@ void Widget::onTransferClicked()
     activeChatroomWidget = nullptr;
 }
 
+bool Widget::confirmExecutableOpen(const QFileInfo file)
+{
+    if (file.isExecutable())
```

Comment:
According to [StackOverflow](http://stackoverflow.com/a/15083959) it should return true on .exe, .com and .bat. On Linux and the likes, it indeed will always return false, but I'm pretty sure they won't ever be executed as programs there anyway? At least, the last time I sent myself a .sh file it simply got opened in my text editor.

I will, however, replace it with the extension list anyway, because I can imagine the odd situation where WINE may execute a .exe despite it not being marked as executable (don't know if it actually does that, though) or something similar. It's probably the safer way to go about it.